resources/views/layouts/base-test.html.twig

```diff
@@ -13,6 +13,8 @@
         {% include "common/head" %}
     {# /head #}
     {# body #}
+        {% include "common/goc" %}
+        {% include "common/alert" %}
```

Comment:
Instead of using the `base-test.html.twig` template on the congrats page (which was meant for testing), we should be using the main `base.html.twig` template. Then you'll also need to add the header block (`{% block header %}{% endblock %}`) [above the body block](https://github.com/GCTC-NTGC/TalentCloud/blob/d15f5946cd2feb05f0b95619f81a9bbd8d122599/resources/views/layouts/base.html.twig#L19) in the `base.html.twig` template.